WarOOP/Models/Warrior.cs

```diff
@@ -17,16 +17,19 @@ public Warrior()
         Attack = 5;
     }
 
-    protected virtual void GetDamageFrom(Warrior enemy)
+    protected internal virtual int GetDamageFrom(Warrior enemy)
     {
         if (IsAlive)
         {
             CurrentHealth -= enemy.Attack;
+            
+            return enemy.CurrentHealth < 0 ? enemy.Attack + enemy.CurrentHealth : enemy.Attack;
```

Comment:
Why do you return the result of this expression `enemy.Attack + enemy.CurrentHealth`?

Reply:
It's necessary when enemy health < 0, in this case it returns only attack down to zero